Source/WebKit/Resources/SandboxProfiles/ios/com.apple.WebKit.Networking.sb.in

```diff
@@ -584,7 +584,10 @@
 
 ;; Access to ContainerManager
 (allow mach-lookup 
-    (global-name "com.apple.containermanagerd"))
+    (global-name
+        "com.apple.containermanagerd"
+        "com.apple.containermanagerd.system"))
```

Comment:
Do we still need to add 'system', even with the code change?

Reply:
Yes, the code change only applies to the WebContent process, not the Network process.

Thanks for reviewing!